src/main/resources/xslt/inputs/ddi/functions.fods

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We still have bit of bad design, as each time you open an fods, with the autosave mechanism, you have a change on inside and insignificant metadatas' file.
These "changes" shouldn't be committed, til we find the correct way to deal with .fods marked unfortunately as modified. They just add complexity and offuscate the real commits.